toolkit/tools/specreader/specreader.go

```diff
@@ -200,8 +244,40 @@ func createChroot(workerTar, buildDir, specsDir, srpmsDir string) (chroot *safec
 	return
 }
 
+func findSpecFiles(specsDir string, parseList map[string]bool) (specFiles []string, err error) {
+	// Find the filepath for each spec in the SPECS directory.
+	if len(parseList) == 0 {
+		specSearch, err := filepath.Abs(filepath.Join(specsDir, "**/*.spec"))
+		if err == nil {
+			specFiles, err = filepath.Glob(specSearch)
+		}
+		if err != nil {
+			logger.Log.Errorf("Failed to find *.spec files. Check that %s is the correct directory. Error: %v", specsDir, err)
```

Comment:
Out of interest, why do you sometimes use `logger.Log.Errorf` to log extra information and in other cases use `fmt.Errorf` to wrap the error?

Reply:
Just very old style. This is the very first go tool we ever built. Wanted to just keep consistent with the existing code here, but generally we prefer wrapping errors with `fmt.Errorf("...%w",err)` these days.